tests/MongoDB.Driver.Tests/AuthenticationTests.cs

```diff
@@ -388,16 +395,19 @@ private BsonDocument CreateX509DatabaseUser(MongoClient client, string userName)
             var createUserCommand =
                 $"{{ createUser : '{userName}'," +
                 @"   roles : [ { role : 'readWrite', db : 'test' },
-                              { role : 'userAdminAnyDatabase', db : 'admin' } 
-                    ]
+                               { role : 'userAdminAnyDatabase', db : 'admin' } ]
+                     writeConcern: { w : 'majority', j : true }
                   }";
             return client.GetDatabase("$external").RunCommand<BsonDocument>(createUserCommand);
         }
 
         private List<object> DropDatabaseUser(MongoClient client, string userName, IEnumerable<string> databases)
         {
-            var dropUserCommand = $"{{ dropUser : '{userName}' }}";
-            
+            var dropUserCommand =
```

Comment:
Can you explain the purpose of a journal option here?

Reply:
I wanted to make sure that the user was definitely created before moving on to the next step: it's strictly speaking not necessary as I later realized the issue was with the backing `Cluster` and lightweight `MongoClients`, and not a race condition between creating the user and authenticating as the user. I think it does no harm to leave the write concern and the journaling option, but I can remove it if you or other reviewers think it's confusing! :)

Reply:
I think it can confuse a bit but at the same time, maybe it's not really important. I will rely on other reviewers here

Reply:
I would remove the journaling options if it contributes nothing to the test.

Reply:
Done